Swiift-project1/Controllers/ViewController.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  ViewController.swift
+//  epam
+//
+//  Created by Kezikov Boris  on 21.01.2020.
+//  Copyright © 2020 Kezikov Boris . All rights reserved.
+//
+
+import UIKit
+
+
+class ViewController: UIViewController, SettingsVCDelegate {
+    private var leftBorder: Int = 0;
+    private var rightBorder: Int = 0;
+    private var gamesPlayedCounter: Int = 0
+    private var magicNumber: Int = 0
+    private static var counter: Int = 0
+    private var settingVC: SettingsViewController!
+    
+    @IBOutlet weak var triesCounter: UILabel!
+    @IBOutlet weak var playButton:UIButton!
+    @IBOutlet var textField: UITextField!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.playButton.setTitle(NSLocalizedString(Constants.play, comment: ""), for: UIControl.State.normal)
+        self.triesCounter.text = NSLocalizedString(Constants.tries,comment: "") + "0"
+        instantiateSwipes()
+        Service.toast(title: NSLocalizedString(Constants.welcomeTitle,comment: ""),
+                      message: NSLocalizedString(Constants.welcomeMessage, comment: ""), vc: self)
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        if (checkParams()){
+            self.magicNumber = Service.generateRandomValue(leftBorder: leftBorder, rightBorder: rightBorder)
+        }
+    }
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        switch segue.identifier {
+        case "StatisticsSegue":
+            guard let dest = segue.destination as? StatisticsViewController else {return}
+            dest.gamesCount = String(gamesPlayedCounter)
+        case "SettingSegue":
+            guard let dest = segue.destination as? SettingsViewController else {return}
+            dest.delegate = self
+        default:
+            break
+        }
+    }
+    
+    @IBAction func readValue(_ sender: Any) {
+        if Service.validateInput(value: textField.text!) {
+            if let text = textField.text, let value = Int(text) {
+                ViewController.counter += 1
+                
+                self.triesCounter.text = NSLocalizedString(Constants.tries,comment: "") + String(ViewController.counter)
+                
+                if value > magicNumber {
+                    
+                    Service.toast(title: NSLocalizedString(Constants.wrongTitle, comment: ""),
```

Comment:
create separate extension file for NSLocalizedString
and use like `Constants.wrongTitle.localized`